src/pages/cli/graphql/examples-and-solutions.mdx

```diff
@@ -564,4 +564,83 @@ query repsByPeriodAndTotal {
     }
   }
 }
-```
\ No newline at end of file
+```
+
+## Advanced
+
+### Batch Put Custom Resolver
+
+Sometimes it's preferred to create objects in bulk, rather than subsequently creating individual objects and waiting for all the requests to complete.
+
+1. Define your schema with a custom mutation (the custom mutation should not be deployed to AppSync beforehand if following these steps)
```

Comment:
I don't understand this part: "(the custom mutation should not be deployed to AppSync beforehand if following these steps)"